test/format/t.c

```diff
@@ -348,7 +348,12 @@ main(int argc, char *argv[])
         wts_open(g.home, &g.wts_conn, true);
         timestamp_init();
     }
-
+    /*
+     * Since RTS is not ran with precise checkpoint, we need to use prepare discover cursor to claim
+     * all pending prepared transactions.
+     */
+    if (GV(PRECISE_CHECKPOINT))
+        wts_prepare_discover(g.wts_conn);
```

Comment:
Also here the convention seems to that the conditionals are within the funciton calls. So the if (precise_checkpoint_enabled) should go in `wts_prepare_discover` (based on other functions in here)?

Reply:
ohh I basically meant moving
```
if (bah)
```
to `wts_prepare_discover`. As based on the convention in the file all the functions seems to do the same.

Reply:
I don't think it's the same lol, but sure no problem, will do

Reply:
done